opensrp-giz-malawi/src/main/java/org/smartregister/giz/presenter/LoginPresenter.java

```diff
@@ -86,12 +92,27 @@ public void processViewCustomizations() {
             }
 
         } catch (Exception e) {
-            Log.d(TAG, e.getMessage());
+            Timber.e(e);
         }
     }
 
     @Override
     public boolean isServerSettingsSet() {
-        return false;
+        try {
+            Setting setting = AncLibrary.getInstance().getContext().allSettings().getSetting(GizConstants.KEY.SITE_CHARACTERISTICS);
```

Comment:
We can later move this functionality to ANC,  it seems this check will and is going to be used by them also on first login in a facility. You can create the issue on ANC and talk with @dubdabasoduba so that he can move the code OR work on it when they get there

Reply:
Yeah i will create an issue and move it

Reply:
https://github.com/OpenSRP/opensrp-client-anc/issues/381